### PR TITLE
AL/MIN-61. Connecting oracles.

### PR DESCRIPTION
**Pull request by lut1k**
**Description:**

1. Added the orml_oracle pallet to runtime.
2. Connected standard pallets pallet_membership, pallet_collective. They are needed for orml_oracle to work.
- pallet_collective :: Instance1 - responsible for the general council on the Minterest blockchain.
- pallet_membership :: Instance1 - manages a members on the general council of the Minterest blockchain.
- pallet_membership :: Instance2 - manages a members in MinterestOracle.
3. Renamed the oracle pallet to module-prices. The data from Oracle cannot be used in business, prices module will do some process and feed prices for Minterest.
4. Implemented the PriceProvider trait in the pallet traits. This trait is responsible for receiving data from oracles.
5. Implemented unit tests for the price module.
6. Fixed the types.json file.

**Changes in storage:**

1. See module prices. Added type LockedPrice - mapping from currency id to it's locked price.
2. See module orml_traits.

**Test scenarios/Use cases:**

Implemented unit-tests for module Prices.

The info will go to the Protocol documentation.

See https://minterestfinance.atlassian.net/wiki/spaces/MINTEREST/pages/134807557/Minterest+API

Module prices: 
1. Extrinsics:
 -  lock_price();
 - unlock_price();
2. Storage query:
 - lockedPrice();

Module orml_oracle:
1. Extrinsics:
 - feed_values();
2. Storage query:
 - RawValues();
3. RPC:
 - oracle_getValue();

### CODE REVIEW

File: pallets/liquidity-pools/src/lib.rs [hunk 1]
```diff
@@ -59,17 +59,17 @@ mod mock;
...
 
-type Oracle<T> = oracle::Module<T>;
 type RateResult = result::Result<Rate, DispatchError>;
 type CurrencyIdResult = result::Result<CurrencyId, DispatchError>;
 type BalanceResult = result::Result<Balance, DispatchError>;
 
 #[frame_support::pallet]
 pub mod module {
 	use super::*;
+	use pallet_traits::PriceProvider;
```
Comment (Apollo50) [inline pallets/liquidity-pools/src/lib.rs:69]: remove import dublication

**Review by Apollo50 — COMMENTED** [review]
Set some initial price for each currency in chain_spec .

File: pallets/liquidity-pools/src/lib.rs (hunk 1 above)
Comment (lut1k) [reply pallets/liquidity-pools/src/lib.rs:69]: Fixed

File: pallets/liquidity-pools/src/lib.rs [hunk 2]
```diff
@@ -434,7 +438,9 @@ impl<T: Config> Pallet<T> {
 				let user_balance_wrapped_tokens = T::MultiCurrency::free_balance(wrapped_id, &who);
 				let user_balance_underlying_asset =
 					Self::convert_from_wrapped(wrapped_id, user_balance_wrapped_tokens).ok()?;
-				let oracle_price = <Oracle<T>>::get_underlying_price(pool_id).ok()?;
+				let oracle_price = T::PriceSource::get_underlying_price(pool_id)
+					.ok_or(Error::<T>::InvalidFeedPrice)
+					.ok()?;
```
Comment (dev-vadim) [inline pallets/liquidity-pools/src/lib.rs:443]: Why do we need Option->Result->Option conversion?
Remove call to ok()

**Review by Pavel-Blaize — APPROVED** [review]

**Pull request merged**